.releaserc

```diff
@@ -0,0 +1,16 @@
+{
+    "branches": [
+        "main",
+        {
+            "name": "next",
+            "prerelease": true
+        }
+    ],
+    "plugins": [
+        "@semantic-release/commit-analyzer",
+        "@semantic-release/release-notes-generator",
+        "@semantic-release/npm",
+        "@semantic-release/github"
+    ],
```

Comment:
Does it create a new release on github with the changelog?

Reply:
Exactly. :)